### PR TITLE
Custom gas price looking like layout for desktop

### PR DESCRIPTION
**Pull request by bconfortin**
https://github.com/augurproject/augur/issues/3825

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🎇

 This link will always point to this PR's latest changes: https://peek.run/m96h5gr

File: packages/augur-ui/src/modules/modal/gas.tsx [hunk 1]
```diff
@@ -105,16 +104,30 @@ export class Gas extends React.Component<GasProps, GasState> {
...
+                onChange={(e) => {
+                  this.updateAmount(parseFloat(e.target.value));
+                }}
+              />
+            </div>
+            <div>
+              <div>
+                <span>&lt; $0.059</span><span> / Trade</span>
+              </div>
+              <span>0.012441 ETH</span>
```
Comment (bthaile) [inline packages/augur-ui/src/modules/modal/gas.tsx:125]: just hardcoding this value for now?

File: packages/augur-ui/src/modules/modal/gas.tsx [hunk 2]
```diff
@@ -105,16 +104,30 @@ export class Gas extends React.Component<GasProps, GasState> {
...
+                type='number'
+                value={this.state.amount}
+                onChange={(e) => {
+                  this.updateAmount(parseFloat(e.target.value));
+                }}
+              />
+            </div>
+            <div>
+              <div>
+                <span>&lt; $0.059</span><span> / Trade</span>
```
Comment (bthaile) [inline packages/augur-ui/src/modules/modal/gas.tsx:123]: just hardcoding this value for now?

File: packages/augur-ui/src/modules/modal/modal.styles.less [hunk 3]
```diff
@@ -1701,6 +1701,70 @@ div.ButtonsRow:last-of-type {
...
+          > span:first-of-type {
+            .mono-10-bold;
+
+            color: #E9E5F2;
+          }
+
+          > span:last-of-type {
+            .text-10-bold;
+
+            color: #848294;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/modal/modal.styles.less]: colors should not be hardcoded. should be using constants

**Review by bthaile — CHANGES_REQUESTED** [review]
use color constants
don't hardcode values in the component, if only temporary values hardcode in container

File: packages/augur-ui/src/modules/modal/gas.tsx (hunk 1 above)
Comment (bconfortin) [reply packages/augur-ui/src/modules/modal/gas.tsx:125]: Yeah. Just doing visuals

File: packages/augur-ui/src/modules/modal/modal.styles.less (hunk 3 above)
Comment (bconfortin) [reply packages/augur-ui/src/modules/modal/modal.styles.less]: Yeah, I was leaving those for the polish PR. As soon as I get the layout done I’ll check and change those, most likely on the next PR, if that’s ok.

Comment (bthaile) [reply packages/augur-ui/src/modules/modal/modal.styles.less]: not going to allow hardcode colors to be merged up when we need to get the color constants worked out. There needs to be a really good reason to merge up hardcoded colors and I don't think this is a good reason.

Comment (bconfortin) [reply packages/augur-ui/src/modules/modal/modal.styles.less]: Understood, won't happen again. Fixed on the latest commit (https://github.com/AugurProject/augur/pull/4501/commits/8ec12168e2ad43cd090f8ac47ba17dad136e3ca4).

Comment (bthaile) [reply packages/augur-ui/src/modules/modal/modal.styles.less]: main reason is so much effort has gone into consolidating and coordinating the color pallet.

**Review by bthaile — APPROVED** [review]
sweet, lgtm

**Pull request merged**